pkg/didcomm/protocol/didexchange/service_test.go

```diff
@@ -517,9 +516,16 @@ func (p *mockProvider) OutboundDispatcher() dispatcher.Outbound {
 	return &mockdispatcher.MockOutbound{}
 }
 
-// CryptoWallet returns the pack wallet service
-func (p *mockProvider) DIDWallet() wallet.DIDCreator {
-	return &mockwallet.CloseableWallet{CreateDid: &did.Doc{
+type mockDIDCreator struct {
```

Comment:
Can all these mockDIDCreator be consolidated in one place? Similar change is being done for DIDExchange mock service in #348 .

Reply:
done